runtime/compiler/env/PersistentCHTable.cpp

```diff
@@ -311,8 +311,7 @@ TR_PersistentCHTable::findClassInfo(TR_OpaqueClassBlock * classId)
 TR_PersistentClassInfo *
 TR_PersistentCHTable::findClassInfoAfterLocking(TR_OpaqueClassBlock *classId,
       TR::Compilation *comp,
-      bool returnClassInfoForAOT,
-      bool validate)
+      bool returnClassInfoForAOT)
```

Comment:
It's not really related to the change you're making in this PR, but would you consider renaming this function and/or the one it calls to make it more clear what the difference is (other than one takes a `TR::Compilation` object and the other takes a `TR_FrontEnd` object) ?